web_responsive/static/src/less/form_view.less

```diff
@@ -1,14 +1,47 @@
 /* Copyright 2016 Ponto Suprimentos Ltda.
+   Copyright 2018 Alexandre Díaz
  * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
 
+@sheet-margin: @sheet-padding;
+@chatter-side-width: @screen-md / 2.5;
+
+// Sided Chatter
+@media (min-width: @screen-md) {
+    .o_chatter_position_sided {
+        .o_form_view {
+            display: flex;
+            height: 100%;
+
+            .o_form_sheet_bg {
+                border-right: 1px solid @table-border-color;
+                overflow: auto;
+                flex: 1 1 auto;
+            }
+
+            .o_chatter {
+                overflow: auto;
+                flex: 0 0 @chatter-side-width;
+
+                .o_chatter_topbar {
+                    height: auto;
+                    flex-wrap: wrap;
+                }
```

Comment:
Inside this block, add this other one:

```less
.o_followers {
    order: -10;
    flex: 2 0 @chatter-side-width;
}
```

That means: *Put this element before the other ones (that default to `order: 0`); then, give him the double than the other ones when growing (because it has 2 buttons), but do not let it shrink smaller than `@chatter-side-width`*

Reply:
![2018-06-20-160030_1366x768_scrot](https://user-images.githubusercontent.com/731270/41663343-833514a4-74a3-11e8-9423-b7ec04e7e6ca.png)

I have better results (if chatter have fixed size) with:
```css
.o_followers {
    order: -10;
    flex: 0 1 @chatter-side-width;
 }
```
Use all the chatter width (no more, but if needed less)
![2018-06-20-155938_1366x768_scrot](https://user-images.githubusercontent.com/731270/41663352-892a31aa-74a3-11e8-830d-7930d82597ed.png)

Reply:
OK, seems legit.